app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -92,6 +92,10 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
                     case "syncing":
                         binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_blue));
                         break;
+                    case "":
```

Comment:
You should not do it based on an empty string. You should check that the Folder object has the paused attribute set to true.

Perhaps model.paused?

Reply:
No. There is a paused attribute on the object returned by the config, that is what should be used.